### PR TITLE
Fix issue #78 - Update README.md

### PR DESCRIPTION
**Pull request by matheusvsantos**
**Description of the change**:

**Reason for the change**:

**Link to original source**:

### Checklist

Make sure all of these items are complete, or else the PR will be ineligible for a code review.

- [x] Code passes all existing [tests](https://github.com/sendgrid/open-source-library-data-collector/tree/master/test)
- [x] Any new functionality added includes new unit tests in [`tests/test.py`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/test/test.py)
- [x] Create or update example code to show the new functionality in action.
- [x] All code, branch, and git naming and style conventions are followed (see [`CONTRIBUTING.md`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/CONTRIBUTING.md#style-guidelines--naming-conventions))
- [x] Feature branch has been rebased off of the latest `master` branch. ( see [`CONTRIBUTING.md`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/CONTRIBUTING.md#creating-a-pull-request) ).

If you have questions, please send an email [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/open-source-library-data-collector?pullRequest=80) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/open-source-library-data-collector?pullRequest=80) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/open-source-library-data-collector?pullRequest=80) it.</sub>

File: README.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
+![SendGrid Logo](https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png)
+
+
 [![Travis Badge](https://travis-ci.org/sendgrid/open-source-library-data-collector.svg?branch=master)](https://travis-ci.org/sendgrid/open-source-library-data-collector)
+[![BuildStatus](https://travis-ci.org/sendgrid/sendgrid-nodejs.svg?branch=master)](https://travis-ci.org/sendgrid/sendgrid-nodejs)
```
Comment (mbernier) [inline README.md]: Can you please change `nodejs` to `open-source-library-data-collector` on these lines?

Comment (matheusvsantos) [reply README.md]: Yeah! I just fix those lines on my last commit https://github.com/sendgrid/open-source-library-data-collector/pull/80/commits/e222a6f1ab8cc76bc20f1a0c4a5a048760c8b36e

File: README.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
+![SendGrid Logo](https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png)
+
+
 [![Travis Badge](https://travis-ci.org/sendgrid/open-source-library-data-collector.svg?branch=master)](https://travis-ci.org/sendgrid/open-source-library-data-collector)
+[![BuildStatus](https://travis-ci.org/sendgrid/open-source-library-data-collector.svg?branch=master)](https://travis-ci.org/sendgrid/open-source-library-data-collector)
```
Comment (mbernier) [inline README.md]: This looks like a duplicate to the line above, can you please remove this one?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+[![Email Notifications Badge](https://dx.sendgrid.com/badge/python)](https://dx.sendgrid.com/newsletter/python)
+[![npm](https://img.shields.io/npm/l/express.svg)]()
```
Comment (mbernier) [inline README.md]: This npm line doesn't apply to this repo, can you please remove it?

**Comment by mbernier** [comment]
I merged in another change and that resolved my comments! Thanks for your help!

**Comment by thinkingserious** [comment]
Hello @matheusvsantos,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**